example/README.md

```diff
@@ -0,0 +1,51 @@
+Example Webpack setup
+======================
+
+Here you'll find an example Webpack configuration. The `webpack.config.js` here will output two bundles here: `application.js` and `vendor.js`, and will process an image.
+
+## Starting
+
+Clone the plugin and run `npm install` in the root folder and in the `example/` folder. Then start `npm run` to start Webpack. In the folder `build/` you should see four files:
```

Comment:
folder**,** and in the `example/`...
"start `npm run` to start" - Not wrong, but maybe not use "start" twice?
"You should now see four files in the `build/` directory." (or folder, whatever)